### PR TITLE
Feature/helm update

### PR DESCRIPTION
**Pull request by ARADDCC013**
helm update
- added stroom
- added app
- updated ingress  (tested with AWS and k8)
- updates logger.ts
- updated values.yaml

### CODE REVIEW

**Comment by ARADDCC013** [comment]
Also updated Chart.yaml as old charts do not exist. Also, tested this config on AWS and Openshift.

File: helm/bailo/templates/bailo/bailo.configmap.yaml [hunk 1]
```diff
@@ -53,6 +67,9 @@ data:
           supportHref: 'mailto:{{ .Values.config.supportEmail }}?subject=Bailo%20Support',
           contactHref: 'mailto:{{ .Values.config.supportEmail }}?subject=Bailo%20Contact',
         },
+        help: {
+          documentationUrl: '{{ .Values.config.documentationLink }}',
```
Comment (a3957273) [inline helm/bailo/templates/bailo/bailo.configmap.yaml:71]: This value doesn't appear to be used in the application anymore.

File: helm/bailo/values.yaml [hunk 2]
```diff
@@ -174,15 +176,16 @@ config:
   s2i:
     image: seldonio/seldon-core-s2i-python37:1.10.0
   supportEmail: "support@example.com"
+  documentationLink: "https://example.com/support"
```
Comment (a3957273) [inline helm/bailo/values.yaml:179]: This value is not used anymore.

File: helm/bailo/values.yaml [hunk 3]
```diff
@@ -174,15 +176,16 @@ config:
...
 
   ui:
     banner:
       enabled: "true"
```
Comment (a3957273) [inline helm/bailo/values.yaml:183]: Let's default the banner to disabled?

**Review by a3957273 — CHANGES_REQUESTED** [review]

File: helm/bailo/templates/bailo/bailo.configmap.yaml (hunk 1 above)
Comment (ARADDCC013) [reply helm/bailo/templates/bailo/bailo.configmap.yaml:71]: removed

File: helm/bailo/values.yaml (hunk 2 above)
Comment (ARADDCC013) [reply helm/bailo/values.yaml:179]: removed

File: helm/bailo/values.yaml (hunk 3 above)
Comment (ARADDCC013) [reply helm/bailo/values.yaml:183]: set to disabled

**Review by a3957273 — APPROVED** [review]

**Pull request merged**